pkg/client/client.go

```diff
@@ -605,41 +655,47 @@ func (d *DataCenterClient) GetGlobalGroupPermissions(ctx context.Context, startP
 		}
 	}
 
-	return permissionsData.GroupsPermissions, page, nil
+	return permissionsData.GroupsPermissions, page, BitbucketError{
+		Error: nil,
+	}
 }
 
-func (d *DataCenterClient) ListGlobalUserPermissions(ctx context.Context, opts PageOptions) ([]UsersPermissions, string, error) {
+func (d *DataCenterClient) ListGlobalUserPermissions(ctx context.Context, opts PageOptions) ([]UsersPermissions, string, BitbucketError) {
 	var nextPageToken string = ""
 	usersPermissions, page, err := d.GetGlobalUserPermissions(ctx,
 		strconv.Itoa(opts.Page),
 		strconv.Itoa(opts.PerPage),
 	)
-	if err != nil {
-		return usersPermissions, "", err
+	if err.Error != nil {
+		return []UsersPermissions{}, "", err
 	}
 
 	if page.HasNext() {
 		nextPageToken = *page.NextPage
 	}
 
-	return usersPermissions, nextPageToken, err
+	return usersPermissions, nextPageToken, BitbucketError{
+		Error: nil,
+	}
 }
 
-func (d *DataCenterClient) ListGlobalGroupPermissions(ctx context.Context, opts PageOptions) ([]GroupsPermissions, string, error) {
+func (d *DataCenterClient) ListGlobalGroupPermissions(ctx context.Context, opts PageOptions) ([]GroupsPermissions, string, BitbucketError) {
 	var nextPageToken string = ""
 	groupsPermissions, page, err := d.GetGlobalGroupPermissions(ctx,
 		strconv.Itoa(opts.Page),
 		strconv.Itoa(opts.PerPage),
 	)
-	if err != nil {
+	if err.Error != nil {
```

Comment:
If there's no error, won't err be nil and cause this to crash?

Reply:
Fixed